spec/index.html

```diff
@@ -251,7 +251,7 @@ <h3>RDF Literals</h3>
       can only be included in a quoted literal using an escape sequence.
       </p>
 
-    <p>The corresponding <a data-cite="RDF12-CONCEPTS#dfn-lexical-form">RDF lexical form</a>
+    <p>The corresponding <a data-cite="RDF12-CONCEPTS#dfn-lexical-form">lexical form</a>
       is the characters between the delimiters, after processing any escape sequences.
       If present, the <a data-cite="RDF12-CONCEPTS#dfn-language-tag">language tag</a>
       is preceded by a '<code>@</code>' (<span class="codepoint">U+0040</span>).
```

Comment:
The `a` should definitely be `an`, as the symbol is pronounced `at`. It's debatable whether the name of the symbol should be provided as well, and if so, whether it should be put into the parentheses.
```suggestion
      is preceded by an At sign, '<code>@</code>' (<span class="codepoint">U+0040</span>).
```

Reply:
We're not entirely consistent on naming such characters in different specs, and if we're going to do so, we should probably have a template.

In Turtle, we have the following different conventions:

* `<code>&lt;</code> (code point <span class="codepoint">U+003C</span>)`
* `'<code>@</code>' (<span class="codepoint">U+0040</span>)`
* `two concatenated <code>^</code> characters, each having the code point <span class="codepoint">U+005E</span>`
* `backslash ('<code>\</code>', <span class="codepoint">U+005C</span>)`
* `<code title="LINE FEED"><sub>LF</sub></code> (<span class="codepoint">U+000A</span>)`
* `LINE FEED characters (U+000A)`

We should be consistent about code point characters and use a span/class combination.

If there is a character name (e.g., from https://en.wikipedia.org/wiki/List_of_Unicode_characters), we should probably use that (although, there's a convention for control characters such as LINE FEED. Given that, we might want to change this sentence to the following:

```suggestion
      is preceded by an  '<code>@</code>' (At sign, <code point <span class="codepoint">U+0040</span>).
```

Reply:
I almost suggested exactly that (but I was lazy and didn't look to see what we'd settled on elsewhere).